tests/model/test_operations.py

```diff
@@ -54,8 +54,8 @@ def make_grid_priors(ages, times, grid_priors, hyper_priors, smooth_id=7):
                     hyper_priors["value"], hyper_priors["dage"], hyper_priors["dtime"], 0],
     ))
     # Make the last one const. If tests want to make a point const, use prior_id=6.
-    prior.loc[6]["lower"] = 0.5
-    prior.loc[6]["upper"] = 0.5
+    prior.loc[6, "lower"] = 0.5
```

Comment:
Oh, gross. That shouldn't have happened.

Reply:
In test code, that I wrote. Yup. No big deal.